x-pack/plugins/reporting/public/share_context_menu/register_pdf_png_reporting.tsx

```diff
@@ -127,6 +127,7 @@ export const reportingScreenshotShareProvider = ({
     };
 
     const isV2Job = isJobV2Params(jobProviderOptions);
+    const requiresSavedState = !isV2Job;
```

Comment:
Are there still ways to access the V1 job export types? If not, then we shouldn't need this conditional.